ci/release/apply_wheel_modifications.sh

```diff
@@ -9,3 +9,7 @@ CUDA_SUFFIX=${2}
 sed -i "s/^version = .*/version = \"${VERSION}\"/g" python/pyproject.toml
 
 sed -i "s/^name = \"rmm\"/name = \"rmm${CUDA_SUFFIX}\"/g" python/pyproject.toml
+
+if [[ $CUDA_SUFFIX == "-cu12" ]]; then
+    sed -i "/cuda-python/ s/>=11.7.1,<12.0/>=12.0,<13.0/g" python/pyproject.toml
```

Comment:
Can we write a sed command that doesn’t depend on the current pinning? Try this (untested):
```suggestion
    sed -i "s/cuda-python.*/cuda-python>=12.0,<13.0/g" python/pyproject.toml
```

Reply:
Done, it was just missing a closing quote and comma